src/odbc_connection.h

```diff
@@ -102,14 +104,13 @@ class ODBCConnection : public Napi::ObjectWrap<ODBCConnection> {
 
   ConnectionOptions connectionOptions;
 
-  SQLSMALLINT maxColumnNameLength;
-  SQLUINTEGER availableIsolationLevels;
+  GetInfoResults    getInfoResults;
 };
 
 Napi::Array process_data_for_napi(Napi::Env env, StatementData *data, Napi::Array napiParameters);
 SQLRETURN bind_buffers(StatementData *data);
 SQLRETURN prepare_for_fetch(StatementData *data);
-SQLRETURN fetch_and_store(StatementData *data);
-SQLRETURN fetch_all_and_store(StatementData *data);
+SQLRETURN fetch_and_store(StatementData *data, bool set_position, GetDataExtensionsSupport get_data_supports, bool *alloc_error);
+SQLRETURN fetch_all_and_store(StatementData *data, bool set_position, GetDataExtensionsSupport get_data_supports, bool *alloc_error);
```

Comment:
Might want to pass get_data_supports by reference instead of copying all the fields, eg using a const reference:

```suggestion
SQLRETURN fetch_and_store(StatementData *data, bool set_position, const GetDataExtensionsSupport& get_data_supports, bool *alloc_error);
SQLRETURN fetch_all_and_store(StatementData *data, bool set_position, const GetDataExtensionsSupport& get_data_supports, bool *alloc_error);
```